### PR TITLE
feat: Add Fish shell completion

### PR DESCRIPTION
**Pull request by jeloba**
### Motivation

Users of the Fish shell would like to have auto completion too.

### Modifications

Added the `pulsarctl completion fish` sub command to generate fish auto completion.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

This change is a trivial rework / code cleanup without any test coverage.

### Documentation

- [x] `doc`

### CODE REVIEW

**Review by nodece — DISMISSED** [review]

File: docs/en/enable_completion.md [hunk 1]
```diff
@@ -94,3 +94,17 @@ pulsarctl completion bash >/usr/local/etc/bash_completion.d/pulsarctl.bash
 echo 'source /usr/local/etc/bash_completion.d/pulsarctl.bash' >> ~/.bashrc
 source ~/.bashrc
 ```
+
+### Fish
+
+To load completions once in your current session run:
+		
+```bash
+pulsarctl completion fish | source
```
Comment (nodece) [inline docs/en/enable_completion.md:103]: This command needs to be corrected.

Log:
```
❯ ./pulsarctl completion fish | source                                   
source: not enough arguments

```

Comment (jeloba) [reply docs/en/enable_completion.md:103]: :thinking: I pushed an update to the PR four days ago, that fixed the issue, but I can have another look.

Comment (jeloba) [reply docs/en/enable_completion.md:103]: The command succeeds with my `pulsarctl` build from the branch. Are you running the Fish shell when executing `pulsarctl completion fish | source`?

Comment (nodece) [reply docs/en/enable_completion.md:103]: > The command succeeds with my `pulsarctl` build from the branch. Are you running the Fish shell when executing `pulsarctl completion fish | source`?

Right.  I'm running this command on zsh.

Comment (jeloba) [reply docs/en/enable_completion.md:103]: That is probably not supported. I guess you would need to execute this in Fish, if you want to test.

Comment (nodece) [reply docs/en/enable_completion.md:103]: Good, it works fine on the fish.

**Review by nodece — DISMISSED** [review]

File: docs/en/enable_completion.md (hunk 1 above)
Comment (nodece) [reply docs/en/enable_completion.md:103]: Could you check the failed CI?

Comment (jeloba) [reply docs/en/enable_completion.md:103]: will do

Comment (jeloba) [reply docs/en/enable_completion.md:103]: I fixed the formatting...

**Review by nodece — APPROVED** [review]

**Pull request merged**